docs/user-guide/helm.md

```diff
@@ -236,8 +236,8 @@ source:
         value: path/to/file.ext
 ```
 
-!!! warning "Reference in multiple sources not supported"
-    Please note that using a multiple sources application will not let you load the file by reference. See [argoproj/argo-cd#13220](https://github.com/argoproj/argo-cd/issues/13220)
+!!! warning "Reference in multiple sources not supported prior to ArgoCD 2.12"
+    Please note that using a multiple sources application will not let you load the file by reference when using ArgoCD versions prior to 2.12
```

Comment:
Looking at this again, I think the whole warning can be removed - the documentation is versioned.

Reply:
Removed.